modules/dnn/test/test_onnx_importer.cpp

```diff
@@ -410,7 +410,8 @@ TEST_P(Test_ONNX_layers, ReduceMean)
 
 TEST_P(Test_ONNX_layers, ReduceSum)
 {
-    testONNXModels("reduce_sum");
+//    testONNXModels("reduce_sum");
```

Comment:
Why is it commented out? As far as I can see, the model is static and the shape of the input is right.

Reply:
Sorry @rogday, it is a mistake. Fixed.